package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "tailwindcss",
   "version": "0.1.3",
-  "description": "A utility-first CSS framework for rapidly building custom user interfaces.",
+  "description":
+    "A utility-first CSS framework for rapidly building custom user interfaces.",
```

Comment:
Maybe it's best to change the `printWidth` to `100` or so, so this one can be on 1 line?